.babelrc

```diff
@@ -0,0 +1,21 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": false
```

Comment:
looks like the default value, any reason to set it explicitly?

Reply:
Oh yes, this should be removed. https://github.com/Open-Attestation/decentralized-renderer-react-components/pull/62/commits/d64efd78da34687f8933fc99766cd511767b1bda